src/client/swagger/unity.yml

```diff
@@ -1162,55 +1168,26 @@ components:
         id:
           description: the idpe id of the user
           type: string
+        # TODO(ariel): find out if these are account or user specific
         firstName:
           type: string
+        # TODO(ariel): find out if these are account or user specific
         lastName:
           type: string
         email:
+          # TODO(ariel): find out about this one
+          description: Not sure if this is the billing account email or the account email, but this might be unnecessary since that should be included in the Account or nested in the Account
           type: string
-        orgId:
-          type: string
-        accountId:
-          type: string
+        account:
+          $ref: '#/components/schemas/Account'
+          description: User Account for the account
```

Comment:
Adding the account here simplifies the need for various meta data that exists as a property of the account.